### PR TITLE
Feature/rails3 add copyright header

### PR DESCRIPTION
**Pull request by tessi**
more copyright headers everywhere!

see: https://www.openproject.org/issues/1902

### CODE REVIEW

File: lib/tasks/copyright.rake [hunk 1]
```diff
@@ -19,6 +19,10 @@ namespace :copyright do
       short_copyright_line("//")
     when :erb
       short_copyright_surrounding("<%#", "#%>")
+    when :rdoc
+      "----------\n#{short_copyright_line(' ')}\n----------\n".gsub(' -- copyright',"==== copyright\n")
+    when :md
+      "----------\n#{short_copyright_line('    ')}\n----------\n".gsub('    -- copyright',"#### copyright\n")
```
Comment (apfelfabrik) [inline lib/tasks/copyright.rake]: Why not use HTML comments as per this suggestion? http://stackoverflow.com/questions/4823468/store-comments-in-markdown-syntax

Comment (tessi) [inline lib/tasks/copyright.rake]: Using HTML comments is MUCH better for .md files. Good catch :)

File: doc/UPGRADING.rdoc [hunk 2]
```diff
@@ -1,3 +1,18 @@
 = Upgrade
 
 The detailed upgrade instructions are located on the {official website}[https://www.openproject.org/projects/support/wiki/Upgrade]
+
+
```
Comment (apfelfabrik) [inline doc/UPGRADING.rdoc:5]: ~~Why at the bottom?~~ _Whatever._

**Comment by apfelfabrik** [comment]
Remaining open questions are summed up here:
https://www.openproject.org/issues/1903

**Pull request merged**